tools/generator/src/PBXProductType+Extensions.swift

```diff
@@ -0,0 +1,38 @@
+import XcodeProj
+
+extension PBXProductType {
+    var fileType: String? {
+        switch self {
+        case .application: return "wrapper.application"
```

Comment:
nit: want to add a comment linking to the source of this mapping?

Reply:
There isn't one. I created each of these targets in Xcode and looked at the product for each and what file type was associated with them.